bennu-portal/src/main/java/org/fenixedu/bennu/portal/domain/MenuItem.java

```diff
@@ -334,4 +343,28 @@ protected User getBennuUser(ApplicationUser appUser) {
         return appUser != null ? User.findByUsername(appUser.getUsername()) : null;
     }
 
+    @Override
+    public Layout getLayoutObject() {
+        return Optional.ofNullable(layoutProviderRegistrySupplier.get())
```

Comment:
@francisco-esteves there's no need for `Optional.ofNullable(layoutProviderRegistrySupplier.get())` because when you call `ServiceProvider.getService(someService.class)`if there's no service available it will fail. 

Which means that the Optional is not doing any sort of safety in this case.